pkg/lendertesting/lend_test.go

```diff
@@ -485,7 +485,7 @@ func (s *lendTestSuite) TestLendTempus() {
 	s.ZcToken.MintReturns(true)
 	s.Env.Blockchain.Commit()
 
-	s.ZcToken.BalanceOfReturns(s.Dep.LenderAddress, big.NewInt(100))
```

Comment:
using the lender contract for a user address?

why not use the s.Env.User* ?

Reply:
or is it a custody thing where `lender` owns it? i bet that's it. nvm... just noticed this was in a `lend` test. i had assumed this was all `redeem` stuff